disable_web_page_preview

gemini1389 opened the pull request:
Added parameter "disable_web_page_preview" for `sendMessage()`.
https://core.telegram.org/bots/api#sendmessage

witnessmenow commented:
Thanks, will go through all PRs in the next couple of days/weeks and merge
all!

On Sun, 31 Oct 2021 at 23:10, Gemini_13 ***@***.***> wrote:

> Added parameter "disable_web_page_preview" for sendMessage().
> https://core.telegram.org/bots/api#sendmessage
> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/witnessmenow/Universal-Arduino-Telegram-Bot/pull/260
> Commit Summary
>
>    - disable_web_page_preview
>    <https://github.com/witnessmenow/Universal-Arduino-Telegram-Bot/pull/260/commits/329c3c73ab829fdaca80acb64d17e6df3442e7c0>
>    - disable_web_page_preview
>    <https://github.com/witnessmenow/Universal-Arduino-Telegram-Bot/pull/260/commits/e12c0dd7adee25ae20f59aec68e94c18a622e46d>
>
> File Changes
>
> (2 files
> <https://github.com/witnessmenow/Universal-Arduino-Telegram-Bot/pull/260/files>
> )
>
>    - *M* src/UniversalTelegramBot.cpp
>    <https://github.com/witnessmenow/Universal-Arduino-Telegram-Bot/pull/260/files#diff-24e4aa549c3bd576e79e17ee6109f8bff6a828f1e06b7e6eb630ee0bf12cf896>
>    (2)
>    - *M* src/UniversalTelegramBot.h
>    <https://github.com/witnessmenow/Universal-Arduino-Telegram-Bot/pull/260/files#diff-16513a0fd85dfbcf76c0f7881833b17dba6088df55713d8e80a0d9fd153b5582>
>    (3)
>
> Patch Links:
>
>    -
>    https://github.com/witnessmenow/Universal-Arduino-Telegram-Bot/pull/260.patch
>    -
>    https://github.com/witnessmenow/Universal-Arduino-Telegram-Bot/pull/260.diff
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/witnessmenow/Universal-Arduino-Telegram-Bot/pull/260>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAL5PQWBNIYMXW5B3JLCYJDUJXEFNANCNFSM5HC2A2CA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>

gemini1389 commented:
Added some more changes:
1. Added a piece of code for `disable_web_page_preview` (not all added last time).
2. Added the `deleteMessage()` function and an example for it.
3. Added the `maxMessageLength` parameter to the constructor (I saw #210 late).
4. Removed `_debug`. There is `TELEGRAM_DEBUG` for debugging.

gemini1389 commented:
@witnessmenow, can you say roughly when you can check PR?
Thanks!
